weave

```diff
@@ -536,20 +538,30 @@ dns_args() {
 }
 
 # Perform operation $1 on local DNS database, for container ID $2 at
-# addresses $3, $4, ...  This function is only called when we know $2
-# is a valid container name
+# addresses $3, $4 ... Uses FQDN set on container and calls tell_dns_fqdn()
```

Comment:
"...for container ID $2, with a FQDN obtained from the container's configured hostname&domain, at addresses $3, $4 ...". No need to mention the tell_dns_fqdn bit.